src/bus.rs

```diff
@@ -253,28 +304,36 @@ where
 
     async fn readn(&mut self, func: u32, addr: u32, len: u32) -> u32 {
         let cmd = cmd_word(READ, INC_ADDR, func, addr, len);
-        let mut buf = [0; 1];
+        let mut buf = [0; 2];
+        let len = if func == FUNC_BACKPLANE { 2 } else { 1 };
```

Comment:
same as above. could you add a comment explaining the +1, now that the comment inside `transaction!` is going away?